docs/extensions/spatial.md

```diff
@@ -90,19 +92,19 @@ Compute relationships and spatial predicates between geometries.
-| BOOLEAN ST_Overlaps(GEOMETRY, GEOMETRY)        | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
-| BOOLEAN ST_Contains(GEOMETRY, GEOMETRY)        | 🧭        | 🔄        | 🔄             | 🦆 or 🔄     | 🔄 (as POLYGON) |
-| BOOLEAN ST_CoveredBy(GEOMETRY, GEOMETRY)       | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
-| BOOLEAN ST_Covers(GEOMETRY, GEOMETRY)          | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
-| BOOLEAN ST_Crosses(GEOMETRY, GEOMETRY)         | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
-| BOOLEAN ST_Difference(GEOMETRY, GEOMETRY)      | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
-| BOOLEAN ST_Disjoint(GEOMETRY, GEOMETRY)        | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
-| BOOLEAN ST_Intersects(GEOMETRY, GEOMETRY)      | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
-| BOOLEAN ST_Equals(GEOMETRY, GEOMETRY)          | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
-| DOUBLE ST_Distance(GEOMETRY, GEOMETRY)         | 🧭        | 🦆 or 🔄   | 🦆 or 🔄        | 🔄          | 🔄 (as POLYGON) |
-| BOOLEAN ST_DWithin(GEOMETRY, GEOMETRY, DOUBLE) | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as POLYGON) |
+| `BOOLEAN ST_Within(GEOMETRY, GEOMETRY)`          | 🧭        | 🦆 or 🔄  | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Touches(GEOMETRY, GEOMETRY)`         | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Overlaps(GEOMETRY, GEOMETRY)`        | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Contains(GEOMETRY, GEOMETRY)`        | 🧭        | 🔄        | 🔄             | 🦆 or 🔄    | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_CoveredBy(GEOMETRY, GEOMETRY)`       | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Covers(GEOMETRY, GEOMETRY)`          | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Crosses(GEOMETRY, GEOMETRY)`         | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Difference(GEOMETRY, GEOMETRY)`      | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Disjoint(GEOMETRY, GEOMETRY)`        | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Intersects(GEOMETRY, GEOMETRY)`      | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_Equals(GEOMETRY, GEOMETRY)`          | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
+| `DOUBLE ST_Distance(GEOMETRY, GEOMETRY)`         | 🧭        | 🦆 or 🔄  | 🦆 or 🔄        | 🔄          | 🔄 (as `POLYGON`) |
+| `BOOLEAN ST_DWithin(GEOMETRY, GEOMETRY, DOUBLE)` | 🧭        | 🔄        | 🔄             | 🔄          | 🔄 (as `POLYGON`) |
```

Comment:
Are these up to date? @Maxxen
Should this switch to be autogenerated from source code somehow?